.bingo/Variables.mk

```diff
@@ -23,6 +23,12 @@ $(BINGO): $(BINGO_DIR)/bingo.mod
 	@echo "(re)installing $(GOBIN)/bingo-v0.3.0"
 	@cd $(BINGO_DIR) && $(GO) build -mod=mod -modfile=bingo.mod -o=$(GOBIN)/bingo-v0.3.0 "github.com/bwplotka/bingo"
 
+BUF := $(GOBIN)/buf-v0.35.1
```

Comment:
Why are we adding buf dependency? Wanted to know for info purpose 😛

Reply:
Because we use now buf as a tool! (: